_data/transport.yml

```
@@ -344,6 +336,14 @@ websites:
     facebook: virginatlantic
     img: virginatlantic.png
 
+  - name: Vy Group
```

Vy Group is the parent company.
```suggestion
  - name: Vy
```